download/es/download-easy.wml

```diff
@@ -19,105 +19,184 @@
+      </div>
+<!-- START BSD -->
+  <div id="openbsd" class="easy bsd">
+    <div class="package" style="padding-top: 13px; border-top: 0px;">
+      <div class="desc">
+        <h2>Navegador Tor para OpenBSD</h2>
+        <p>These are installation instructions for running Tor Browser in a OpenBSD environment.</p>
+        <p>To install from OpenBSD's packages, run:</p>
+        <pre>pkg_add tor-browser</pre>
+        <p>Sometimes the most recent version of Tor Browser on OpenBSD is behind the current release. The available version of TB on OpenBSD should be checked with:
+        <pre>pkg_info -Q tor-browser</pre>
+        <p>If provided version is not the current Tor Browser version, it is not recommended.</p>
+        <p>To install the Tor Browser port from an updated ports tree, run:</p>
+        <pre>cd /usr/ports/meta/tor-browser && make install</pre>
       </div>
+    </div>
+  </div>
+<!-- START ANDROID -->
+  <div id="android" class="easy android">
+    <div class="package" style="padding-top: 13px; border-top: 0px;">
+      <div class="desc">
+          <h2>Navegador Tor para Android</h2>
+          <p>Tor Browser for Android has only be released in alpha-version. <br />
+          You can download the alpha release on <a href="https://play.google.com/store/apps/details?id=org.torproject.torbrowser_alpha">GooglePlay</a> or from
+          our <a href="https://www.torproject.org/projects/torbrowser.html.en#downloads-alpha">alpha download list</a>.
+          </p>
+<!--
+          <p>
+          Introducing Tor Browser for Android, the mobile browser with the
+          highest privacy protections ever available and on par with Tor
+          Browser for desktop. You can download the alpha release on
+          <a href="https://play.google.com/store/apps/details?id=org.torproject.android">GooglePlay</a>
+          or <a href="">FDroid</a>, or you can get the apk by clicking on the
+          button below.
+          </p>
+          <p>
+          Note: For this release, you also need to install Orbot, you can
+          learn more about it
+          <a href="https://www.torproject.org/docs/android.html.en">here</a>.
+          </p>
+          <p>
+          Known issue: Our Security Slider is now under ‘Security Settings,
+          but because of a small issue, it’s only showing up after you restart
+          the app. We plan on fixing it for the next release.
+          </p>
+          <form class="downloads">
+            <a class="button" href="../dist/torbrowser/mobile/<version-torbrowserbundleandroid>/tor-browser-android-arm-<version-torbrowserbundleandroid>.apk"><span class="strong">Download</span><span class="normal">Android</span></a>
+            <div class="sig">
+              (<a class="android-tbb-sig" href="../dist/torbrowser/mobile/<version-torbrowserbundleandroid>/TorBrowser-<version-torbrowserbundleandroid>.asc">sig</a>)
+              <a class="siginfo" href="<page docs/verifying-signatures>">What's This?</a>
```

Comment:
This part has been updated by 73d98b306 and needs to be rebased and translated.

Are we ok to uncomment the android section (also on the english download-easy page)?